src/main/java/sirius/biz/model/LoginData.java

```diff
@@ -205,13 +200,38 @@ protected void autofill() {
 
     /**
      * Clears all internal fields so that a new password will be generated when the underlying entity is saved.
+     *
+     * @deprecated use {@link LoginData#forceGenerationOfPassword(String)} instead
      */
+    @Deprecated
```

Comment:
nope - keep this method instead of forcing  forceGenerationOfPassword(null) as this feels unnatural